File: LuceneAdventureWorks/Search/AnalyzerFactory.cs
using Lucene.Net.Analysis;
using Lucene.Net.Analysis.Standard;
using Lucene.Net.Util;

namespace LuceneAdventureWorks.Search;

public static class AnalyzerFactory
{
    public static Analyzer Create()
    {
        // TODO: support configurable analyzer types
        return new StandardAnalyzer(LuceneVersion.LUCENE_48);
    }
}

File: java/LuceneAdventureWorks/src/main/java/org/example/AnalyzerFactory.java
package org.example;

import org.apache.lucene.analysis.Analyzer;
import org.apache.lucene.analysis.standard.StandardAnalyzer;
import org.apache.lucene.util.Version;

public final class AnalyzerFactory {
    public static Analyzer Create() {
        // TODO: support configurable analyzer types
        return new StandardAnalyzer(Version.LUCENE_48);
    }
}

